### ISSUE TITLE
Not compliant with eIDAS 2.0 unlinkability. No support for pseudonym PID or trustworthy identity

### ISSUE THREAD

**Issue by OBIvision** [eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework#305]
### Description
Digital society basically works better when services DO NOT create personal data. This is reflected in the eIDAS 2.0 unlinkability requirement implementing trustworthy anonymity with or without accountability (conditional identification or trustworthy means for de-anonymization)

### User Story
Citizens can create a new trustworthy anonymous or trustworthy pseudonym identity as root for a specific transaction 
Digital services - both public and private sector - can with support from data portability function without creating personal data and without issuers being forced to identify linkable,.

### Acceptance Criteria
Strong Trustworthy anonymity e.g. for research and basis for customization without violating unlinkablity.

### Priority
Essential. Without this the technical implementation will be non-compliant with both eIDAS 2.0 and GDPR

### Technical Notes and Constraints
This can technically be achieved in many ways. Problem is that the present interfaces block security'

**Comment by digeorgi** [comment]
Thank you for your comment. If we understand correctly, you propose to add the possibility for the Wallet Instance to create pseudonyms. This will in fact be added to ARF 1.5.0 by requiring Wallets to support FIDO passkeys. Please let us know if that answers your comment.

**Comment by OBIvision** [comment]
> add the possibility for the Wallet Instance to create pseudonyms. This will in fact be added to ARF 1.5.0 by requiring Wallets to support FIDO passkeys. 

First, yes we are talking true unlinkable pseudonyms enabling full and all aspects of transactions - starting with qualified unlinkable anonymous pseudonyms and ending when GDPR begin applying as trustworthy unlinkability has been broken (all parties included).

Notice here the systemic failure of ARF use cases to even begin addressing the issues. Every single use case described are inherently non-compliant by design as state-of-the-art have made linkable identification unnecessary and thus illegal for the purpose.

Second. FIDO passkeys are just more BigTech surveillance by design - part of the problem, not the solution.

**Comment by OBIvision** [comment]
> Thank you for your comment. If we understand correctly, you propose to add the possibility for the Wallet Instance to create pseudonyms. This will in fact be added to ARF 1.5.0 by requiring Wallets to support FIDO passkeys. Please let us know if that answers your comment.

No. The added is not quallified unlinkable, but merely some randomly token-linked pseudonym - that in addition to not take all the side-channels into account (e.g. device fingerpriniting).

A qualified unlinkable signature as to be able to act as an anchor for an entire transaction.

**Comment by OBIvision** [comment]
> Closed after release of ARF 1.10.0

That is a claim without documentation
